app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx

```diff
@@ -178,13 +178,13 @@ export class Widget extends Component {
   };
 
   fetchWidget = (params = {}) => {
-    const { tracking, isFullscreen } = this.props;
+    const { tracking, isFullscreen, widgetType } = this.props;
     const url = this.getWidgetUrl(params);
 
     clearTimeout(this.silentUpdaterId);
     tracking.trackEvent(DASHBOARD_PAGE_EVENTS.REFRESH_WIDGET);
 
-    if (!isWidgetDataAvailable(this.state.widget)) {
+    if (!isWidgetDataAvailable(this.state.widget) || widgetType === MOST_POPULAR_PATTERNS) {
```

Comment:
This component shouldn't know about any widgets. I suggest adding a new boolean argument `force` instead